<?xml version="1.0" encoding="utf-8"?>

<launch>
  <arg name="debug" default="false" />
  <arg name="rviz_sim" default="1" />
  <arg name="robot"  default="full_v2"/>
  <arg name="gzpose" default="-x -0 -y 0 -z 1.0 -R 0.0 -P 0.0 -Y 0.0"/>
  <arg name="world"  default="saved_world"/>
  <arg name="gazebo" default="true" />


<include file="$(find talos_gazebo)/launch/talos_gazebo.launch">
	<arg name="world" value= "$(arg world)"/>
	<arg name="gui" value="$(arg gazebo)"/>
	<arg name="gzpose" value="$(arg gzpose)"/>
</include>


</launch>
